sklearn/neural_network/tests/test_base.py

```diff
@@ -27,3 +27,71 @@ def test_log_loss_1_prob_finite(y_true, y_prob):
     # y_proba is equal to 1 should result in a finite logloss
     loss = log_loss(y_true, y_prob)
     assert np.isfinite(loss)
+
+
+def test_squared_loss():
+    y_true = np.array([[0, 1], [0, 0]])
+    y_pred = np.array([[1.0, 1.0], [4.0, 0.0]])
+
+    assert squared_loss(y_true, y_pred) == pytest.approx(2.125)
+
+    sample_weight = np.array([1, 2])
+    assert squared_loss(y_true, y_pred, sample_weight) == pytest.approx(2.75)
+
+
+def test_log_loss():
+    y_true = np.array([[1, 0, 0], [0, 1, 0]])
+    y_prob = np.exp([[-0.2, -2, -2], [-1, -0.1, -1]])
+
+    assert log_loss(y_true, y_prob) == pytest.approx(0.15)
+
+    sample_weight = np.array([1, 4])
+    assert log_loss(y_true, y_prob, sample_weight) == pytest.approx(0.12)
```

Comment:
Can a short comment be added to specify where the values like 2.125 and 2.75 come from just to make it clear?

Reply:
Thanks for your review! I was trying to come up with test data that could get nicer numbers, based on a previous comment https://github.com/scikit-learn/scikit-learn/pull/25646#discussion_r1612173142

Reply:
@OmarManzoor Do you think a comment is still needed here?

Reply:
I think since we are just computing values and checking for them I think this is fine then. Thanks!